src/Workspaces/Core/Portable/FindSymbols/FindReferences/Finders/ParameterSymbolReferenceFinder.cs

```diff
@@ -31,7 +31,7 @@ protected override bool CanFind(IParameterSymbol symbol)
             // elsewhere as "paramName:" or "paramName:=".  We can narrow the search by
             // filtering down to matches of that form.  For now we just return any document
             // that references something with this name.
-            return FindDocumentsAsync(project, documents, cancellationToken, symbol.Name);
+            return FindDocumentsAsync(project, documents, SupportsGlobalSuppression(symbol), cancellationToken, symbol.Name);
```

Comment:
were there parameter tests?

Reply:
They are not supported in global suppressions. However, I did find that current logic does not find named type references for parameter types in doc comment IDs. For example `M:C.X(C)` for a method `X` defined in type `C` only finds first `C` reference, but not the reference to `C` in parameter list. I'll fix that with a test.

Reply:
Actually, investigating a bit more, handling references to types in parameter lists is not trivial as it can have constructed generic types. I am going to file a follow-up issue to track this case.

Reply:
https://github.com/dotnet/roslyn/issues/44401